radixdlt/src/main/java/org/radix/utils/Duration.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package org.radix.utils;

```

Comment:
We are in the process of moving all code into com.radixdlt. In particular, the org.radix stuff doesn't have checkstyle applied, as a fair bit of it is historical, and not always current coding standards.

Please move this into com.radixdlt.utils.

Reply:
Done